boards/heltec_mesh_pocket.json

```diff
@@ -0,0 +1,54 @@
+{
+    "build": {
+      "arduino": {
+        "ldscript": "nrf52840_s140_v6.ld"
+      },
+      "core": "nRF5",
+      "cpu": "cortex-m4",
+      "extra_flags": "-DNRF52840_XXAA",
+      "f_cpu": "64000000L",
+      "hwids": [
+        ["0x239A", "0x4405"],
+        ["0x239A", "0x0029"],
+        ["0x239A", "0x002A"]
+      ],
+      "usb_product": "HT-n5262",
+      "mcu": "nrf52840",
+      "variant": "heltec_mesh_node_t114",
```

Comment:
should this be "variant": "heltec_mesh_pocket" or similar?

Reply:
Oh, I'm so sorry about this. It was a careless mistake that has already been fixed.